spec/GenericSpec.hs

```diff
@@ -170,6 +170,11 @@ spec = do
     it "is False if there is no usage" $ do
       uses (named "m") (EntryPoint "main" (Reference "f")) `shouldBe` False
 
+    it "works with RecordUpdate" $ do
+      uses (named "f") (RecordUpdate (Reference "f") [("g", Reference "h")]) `shouldBe` True
+      uses (named "h") (RecordUpdate (Reference "f") [("g", Reference "h")]) `shouldBe` True
+      uses (named "g") (RecordUpdate (Reference "f") [("g", Reference "h")]) `shouldBe` False
```

Comment:
Not married to this. See description.

Reply:
I am unsure, too. 

I would take the same path as with assignments. If we are considering assignment as usage, then should we consider record update's assignee as a usage too